hello-slack/app.py

```
@@ -0,0 +1,378 @@
+# もし hello-slack/ と local-transcriber/ が同じ階層にあるなら...
+current_dir = os.path.dirname(os.path.abspath(__file__))
+parent_dir = os.path.dirname(current_dir)
+local_transcriber_path = os.path.join(parent_dir, 'local-transcriber')
+if local_transcriber_path not in sys.path:
+    sys.path.append(local_transcriber_path)
+
+
+try:
+    from transcription import convert_mp3_to_wav, transcribe_with_openai, diarize_with_resemblyzer
+except ImportError as e:
+    logging.error(f"Failed to import from local-transcriber: {e}. Ensure the module exists at {local_transcriber_path} and path is correct.")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)